build/build.proj

```diff
@@ -116,7 +116,7 @@
     <PropertyGroup>
       <TestProjectPaths>$(RepositoryRootDirectory)test\NuGet.Tests.Apex\NuGet.Tests.Apex\NuGet.Tests.Apex.csproj</TestProjectPaths>
       <TestResultsFileName>ApexTestsVS15</TestResultsFileName>
-      <TestVisualStudioVersion>$(VisualStudioVersion)</TestVisualStudioVersion>
```

Comment:
Why did you revert this?

Reply:
I thought that this breaks the APEX tests on the CI. Viewing the before and after builds of this change, it doesn't show any difference between those builds.

Reply:
Add it back.

Reply:
Actually remove all TestVisualStudioVersion completely. 

They add no value after you make that change.